BLSQ/iaso
# IA-3146 orgunit default image

Be able to choose one of the image uploaded with a submission and set it as default for a specific org unit

Related JIRA tickets : IA-3146

## Self proofreading checklist



## Changes

- adapt model to allow an instanceFile has default_image + test
- adapt org unit api to return default_image + Tests
- change get attachment method to be able to return only images, returning also mime type
- add a new tab in org unit details to display all possible images
- allow user to select one of the picture to be the default one
- modify update method to save default_image
 
## How to test

Find an existing instance and make sure it is linked to an org unit using the admin
Create a new instance file using an image and link it to this instance
Repeat the process a few times
Visit the detail page of this org unit, click on images and play with the favorite button, reload the page to see if changes has been saved

## Print screen / video

https://github.com/user-attachments/assets/3d798cb6-d179-40a9-be0e-ba81e3a0230e

![Screenshot 2024-08-29 at 14 41 37](https://github.com/user-attachments/assets/34b40a34-b415-4e15-843b-5eff24fb16e9)


## Notes
Setuper should probably use a few images while creating org units
